Convex_hull_3/doc/Convex_hull_3/CGAL/convex_hull_3.h

```diff
@@ -65,4 +65,35 @@ void convex_hull_3(InputIterator first, InputIterator last,
 Object& ch_object,
 const Traits& ch_traits = Default_traits);
 
+
+/*!
+\ingroup PkgConvexHull3Functions
+
+\brief computes the convex hull of the set of points in the range.
+The result is put in an output iterator as a list of points.
```

Comment:
I'd rephrase as something like:  put in `out` the point from `range` that are on its convex hull.